### ISSUE TITLE
Updating the documentation for the usage of logger during the run

### ISSUE THREAD

**Issue by shabazpatel** [datmo/datmo#258]
example usage,
```
from datmo.logger import Logger

datmo_logger = Logger()
config = {'a': 1}
datmo_logger.log_config(config)
# Perform analysis to get results

result = {'a': 1}
datmo_logger.log_result(result)
```

**Comment by asampat3090** [comment]
This should be updated via the docstrings for the functions that d efine them. The docs will automatically pull in the information

**Comment by asampat3090** [comment]
merged into #287
